lemarche/tenders/migrations/0094_add_templatetransactional_tender_author_commercial_partners.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+
+
+def create_template(apps, schema_editor):
+    TemplateTransactional = apps.get_model("conversations", "TemplateTransactional")
+    TemplateTransactional.objects.create(
+        name="Dépôt de besoin : auteur : notification quand son besoin a été validé et pris en charge par les partenaires commerciaux",
+        code="TENDERS_AUTHOR_CONFIRMATION_VALIDATED_COMMERCIAL_PARTNERS",
+        description="Envoyé à l'auteur du besoin lorsque son besoin a été validé et qu'il n'est pas envoyé aux structures mais proposé aux partenaires commerciaux",
+    )
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("tenders", "0093_alter_tender_send_to_commercial_partners_only"),
+    ]
+
+    operations = [
+        migrations.RunPython(create_template),
```

Comment:
```suggestion
        migrations.RunPython(create_template, reverse_code=migrations.RunPython.noop),
```
ça permet de rollback plus facilement la migration si besoin

Reply:
ah oui, bien vu, il faudrait même supprimer le template d'ailleurs